daffodil-lib/src/test/scala/edu/illinois/ncsa/daffodil/util/TestDecimalUtils.scala

```diff
@@ -0,0 +1,1293 @@
+/*
```

Comment:
Did you make up the data for these tests themselves or were you able to find a good resource for tests? Nice amount of tests!

Reply:
I made up the data for these tests.  I did some searching online and about the only thing I found were some other people's github repos that also implemented PackedDecimal and a few of them had some tests, but none of them were very comprehensive.